arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -0,0 +1,2119 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to delete a service diagnostic setting"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "ServiceDiagnosticSettings"
+        ],
+        "operationId": "ServiceDiagnosticSettings_Get",
+        "description": "Gets the active diagnostic settings. To get the diagnostic settings being applied, use GetStatus.",
+        "parameters": [
+          {
+            "name": "resourceUri",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource identifier of the configuration."
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-07-01",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get one service diagnostic setting",
+            "schema": {
+              "$ref": "#/definitions/ServiceDiagnosticSettingsResource"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ServiceDiagnosticSettings"
+        ],
+        "operationId": "ServiceDiagnosticSettings_Put",
+        "description": "Create or update new diagnostic settings for the specified resource. This operation is long running. Use GetStatus to check the status of this operation.",
+        "parameters": [
+          {
+            "name": "resourceUri",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource identifier of the configuration."
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-07-01",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServiceDiagnosticSettingsPutParameters"
+            },
+            "description": "Parameters supplied to the operation."
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created service diagnostic setting"
+          },
+          "200": {
+            "description": "Successful request to create a service diagnostic setting"
+          },
+          "202": {
+            "description": "Accepted request to create a service diagnostic setting"
+          }
+        }
```

Comment:
I am checking if any of our operations is longrunning or is it just that we do not need to get anything from the response (from PUT, PATCH, DELETE)